risc0/zkvm/Cargo.toml

```diff
@@ -171,6 +171,7 @@ prove = [
   "risc0-circuit-recursion/prove",
   "risc0-circuit-rv32im/prove",
   "risc0-zkp/prove",
+  "risc0-binfmt/recursion",
```

Comment:
Can you sort this?

Reply:
Fixed in 22b7e10e68ae473ad62ea714655663feded17bef